oereb_gewaesserschutz/build.gradle

```diff
@@ -179,7 +182,28 @@ task importDataToLive(type: Ili2pgReplace) {
     dbschema = "live"
     dataFile = file(Paths.get("$rootDir/" + xtfFileName))
     dataset = groundwaterProtectionDataSet
-    disableValidation = true 
+    disableValidation = true
+    
+}
+
+task zipXtfFile(type: Zip) {
```

Comment:
Ich würde die beiden neuen Task hinter den refreshOerebWMSTablesLive platzieren, weil der Import in die DB und die Aktualisierung der WMS-Tabellen sehr eng zusammenhängen, der Upload aber wieder etwas ganz Anderes ist.